src/search.cpp

```diff
@@ -206,6 +206,16 @@ void MainThread::search() {
   }
   else
   {
+      int rootComplexity;
+      bestValue = -VALUE_INFINITE;
```

Comment:
This line is added due to lazy skip in classical evaluation. (valgrind fails without this)
I'd like for anyone to confirm that this part is non-functional.